packages/app-content-pages/server/server.js

```diff
@@ -2,24 +2,31 @@ if (process.env.NEWRELIC_LICENSE_KEY) {
   require('newrelic')
 }
 
-const http = require('http')
+const express = require('express')
+const morgan = require('morgan')
 const next = require('next')
 
-const configureNextServer = require('./configure-next-server')
+const setAssetPrefix = require('./set-asset-prefix')
+const setCacheHeaders = require('./set-cache-headers')
 
+const port = parseInt(process.env.PORT, 10) || 3000
 const dev = process.env.NODE_ENV !== 'production'
 const app = next({ dev })
-const port = parseInt(process.env.PORT, 10) || 3000
+const handle = app.getRequestHandler()
+
+setAssetPrefix(app)
 
 app.prepare().then(() => {
-  const server = new http.Server((req, res) =>
-    configureNextServer(app, req, res))
+  const server = express()
+  server.use(morgan('combined'))
 
-  server.listen(port, err => {
-    if (err) {
-      throw err
-    }
+  server.get('*', (req, res) => {
```

Comment:
Minor not blocking: Does this server ever have to handle other HTTP verbs? e.g. OPTIONS for cors?

Reply:
Not afaik...